src/Pdp/Exception/SeriouslyMalformedUrlException.php

```diff
@@ -1,13 +1,14 @@
 <?php
-
 /**
- * PHP Domain Parser: Public Suffix List based URL parsing.
+ * Public Suffix List PHP: Public Suffix List based URL parsing.
```

Comment:
These should all change back to "PHP Domain Parser: Public Suffix List based URL parsing."

Reply:
(I used "Public Suffix List PHP" for the new library.)

Reply:
yep .. I'll updatethat